README.md

```diff
@@ -39,6 +40,16 @@ List of links to the various checkers out there on the web for sites, domains, s
 | https://saucelabs.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 | https://testingbot.com | Test apps and websites in many browsers. Can be integrated in CI workflows. | Free trial, free for open source projects, paid |
 
+## Cron Jobs 🌎
+
+| URL | Description | 💳 |
+| --- | ----------- | -- |
+| https://healthchecks.io/ | Simple and Effective Cron Job Monitoring. | Free |
```

Comment:
Can these be put into alphabetical order? Both sections and links within them should be. Thanks!

https://github.com/Brunty/awesome-checker-services/blob/master/CONTRIBUTING.md#getting-started